tests/integration/tier1/config-file/test_config_file.py

```diff
@@ -21,111 +21,9 @@ def remove_files(config):
-    config = [Config("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password")]
-    create_files(config)
-
-    with convert2rhel(("--no-rpm-va --password password --debug")) as c2r:
-        c2r.expect("DEBUG - Found password in /root/.convert2rhel.ini")
-        c2r.expect(
-            "WARNING - You have passed either the RHSM password or activation key through both the command line and"
-            " the configuration file. We're going to use the command line values."
-        )
-        c2r.sendcontrol("c")
-    assert c2r.exitstatus != 0
-
-    remove_files(config)
-
-
-def test_user_path_pswd_file_priority(convert2rhel):
-    config = [
-        Config("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
-        Config("~/password_file", "file_password"),
-    ]
-    create_files(config)
-
-    with convert2rhel(('--no-rpm-va -f "~/password_file" --debug')) as c2r:
-        c2r.expect("DEBUG - Found password in /root/.convert2rhel.ini")
-        c2r.expect("WARNING - Deprecated. Use -c | --config-file instead.")
-        c2r.expect(
-            "WARNING - You have passed the RHSM credentials both through a config file and through a password file."
-            " We're going to use the password file."
-        )
-        c2r.sendcontrol("c")
-    assert c2r.exitstatus != 0
-
-    remove_files(config)
-
-
-def test_std_paths_priority_diff_methods(convert2rhel):
-    config = [
-        Config("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
-        Config("/etc/convert2rhel.ini", "[subscription_manager]\nactivation_key = config2_activationkey"),
-    ]
-    create_files(config)
-
-    with convert2rhel("--no-rpm-va --debug") as c2r:
-        c2r.expect("DEBUG - Found password in /root/.convert2rhel.ini")
-        c2r.expect("DEBUG - Found activation_key in /etc/convert2rhel.ini")
-        c2r.expect(
-            "WARNING - Passing the RHSM password or activation key through the --activationkey or --password options"
-            " is insecure as it leaks the values through the list of running processes."
-            " We recommend using the safer --config-file option instead."
-        )
-        c2r.expect(
-            "WARNING - Either a password or an activation key can be used for system registration."
-            " We're going to use the activation key."
-        )
-        c2r.sendcontrol("c")
-    assert c2r.exitstatus != 0
-
-    remove_files(config)
-
-
-def test_std_paths_priority(convert2rhel):
-    config = [
-        Config("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
-        Config("/etc/convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
-    ]
-    create_files(config)
-
-    with convert2rhel("--no-rpm-va --debug") as c2r:
-        c2r.expect("DEBUG - Found password in /root/.convert2rhel.ini")
-        c2r.sendcontrol("c")
-    assert c2r.exitstatus != 0
-
-    remove_files(config)
```

Comment:
Why it was all removed?

Reply:
I have moved the non-destructive part to tier0 
See https://github.com/oamg/convert2rhel/blob/4b97a984319328221d080e9a54ec7d698216b524/tests/integration/tier0/config-file/test_config_file.py

Reply:
cool. makes sense to me. Thanks!